src/getHumanAge.test.js

```diff
@@ -4,7 +4,38 @@ describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test('should return [0, 0] for ages [0, 0]', () => {
```

Comment:
its difficult to understand from the title what this test is about 
should return 0 for cat and dog when they are less than 15 years old - as an example